### PR TITLE
Sane camera clipping

### PR DESCRIPTION
**Pull request by zeroZshadow**
#### Description
Added sane camera clipping and disabled the `TileEnables`.
They need to removed later when the Demo code is merged in.

#### Tests
Tested on Samsung S9 and PC

#### Documentation
- Just an internal change

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 ### Internal
 
 - Fixed package dependencies.
+- Disable TileEnabler functionality. Replaced by sane camera clipping.
```
Comment (paulbalaji) [inline CHANGELOG.md]:
```suggestion
- Replaced `TileEnabler` functionality with sane camera clipping.
```

File: workers/unity/Assets/Fps/Scripts/SetupLogic/CameraClipQuality.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using Fps;
+using UnityEngine;
+
+public class CameraClipQuality : MonoBehaviour
+{
+    [SerializeField] private float checkoutBias = 10.0f;
+
+    void Awake()
+    {
+        var camera = GetComponent<Camera>();
```
Comment (paulbalaji) [inline workers/unity/Assets/Fps/Scripts/SetupLogic/CameraClipQuality.cs]: worth adding `[RequireComponent(typeof(Camera))]`?

File: workers/unity/Assets/Fps/Scripts/GameLogic/SkyboxCamera.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿using UnityEngine;
+
+public class SkyboxCamera : MonoBehaviour
```
Comment (jessicafalk) [inline workers/unity/Assets/Fps/Scripts/GameLogic/SkyboxCamera.cs]: why is this needed?

Comment (zeroZshadow) [reply workers/unity/Assets/Fps/Scripts/GameLogic/SkyboxCamera.cs]: Since there are now 2 cameras. 1 rendering just the sky and environment, and 1 rendering the game itself.
This script syncs the rotation and fov of the game camera, so that it feels like its 1 space.

Comment (jamiebrynes7) [inline workers/unity/Assets/Fps/Scripts/GameLogic/SkyboxCamera.cs]: Needs a namespace

File: workers/unity/Assets/Fps/Scripts/GameLogic/SkyboxCamera.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+{
+    private Camera myCamera;
+
+    private void Awake()
+    {
+        myCamera = GetComponent<Camera>();
+    }
+
+    // Update is called once per frame
+    void LateUpdate()
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Fps/Scripts/GameLogic/SkyboxCamera.cs]: Access modifier please

File: workers/unity/Assets/Fps/Scripts/GameLogic/SkyboxCamera.cs [hunk 5]
```diff
@@ -0,0 +1,24 @@
+﻿using UnityEngine;
+
+public class SkyboxCamera : MonoBehaviour
+{
+    private Camera myCamera;
+
+    private void Awake()
+    {
+        myCamera = GetComponent<Camera>();
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Fps/Scripts/GameLogic/SkyboxCamera.cs]: Should we put a `[RequireComponent()]` annotation?

File: workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileEnabler.cs [hunk 6]
```diff
@@ -6,61 +6,19 @@ public class TileEnabler : MonoBehaviour
...
 
         private float CheckoutDistanceSquared;
 
-        private MeshRenderer meshRenderer;
-
         public void Initialize(bool isClient)
         {
             if (!isClient)
             {
                 Destroy(this);
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileEnabler.cs]: So what does this script do now?

File: workers/unity/Assets/Fps/Scripts/SetupLogic/CameraClipQuality.cs [hunk 7]
```diff
@@ -0,0 +1,14 @@
+﻿using Fps;
+using UnityEngine;
+
+[RequireComponent(typeof(Camera))]
+public class CameraClipQuality : MonoBehaviour
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Fps/Scripts/SetupLogic/CameraClipQuality.cs]: Needs a namespace

File: workers/unity/Assets/Fps/Scripts/SetupLogic/CameraClipQuality.cs [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+{
+    [SerializeField] private float checkoutBias = 10.0f;
+
+    void Awake()
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Fps/Scripts/SetupLogic/CameraClipQuality.cs]: Needs an access modifier

File: workers/unity/Assets/Fps/Scripts/SetupLogic/MobileWorkerConnectorBase.cs [hunk 9]
```diff
@@ -127,8 +127,8 @@ protected override void HandleWorkerConnectionEstablished()
         {
             var world = Worker.World;
 
-            // Only take the Heartbeat from the PlayerLifecycleConfig Client Systems.
-            world.GetOrCreateManager<HandlePlayerHeartbeatRequestSystem>();
+            PlayerLifecycleHelper.AddClientSystems(world, autoRequestPlayerCreation: false);
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Fps/Scripts/SetupLogic/MobileWorkerConnectorBase.cs]: Nice driveby :)

File: workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileEnabler.cs (hunk 6 above)
Comment (zeroZshadow) [reply workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileEnabler.cs]: Nothing, until Jess's stuff is merged in. Removing this now will require changes in the worker connectors and handler, so I want to avoid those merge conflicts.
